http_api.go

```diff
@@ -0,0 +1,168 @@
+package worker
+
+import (
+	"crypto/subtle"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+)
+
+// APIHandler handles requests to worker's HTTP API.
+type APIHandler struct {
```

Comment:
This is definitely a nitpick, but I think that this should be changed to `HTTPAPI` or `HttpApi` or the file name should be `api_handler.go` (or something else?)

Reply:
Yeah, I think I almost called it `HTTPAPI` and balked because of the EXCESSIVE CAPS, but it's probably a better name for it even so.

Reply:
Maybe `WebApp`/`web_app.go`?